api/metrics/v1/rules.go

```diff
@@ -12,6 +13,29 @@ import (
 	"github.com/observatorium/api/rules"
 )
 
+func marshalRules(rawRules rules.Rules) ([]byte, error) {
+	body, err := yaml.Marshal(rawRules)
```

Comment:
as a rule, we try to avoid defining functions that only call one function on the inside. As it stands there is no benefit for the caller to call `marshalRules` over directly using `yaml.Marshal`

Reply:
Sure, let me change!